tutorials/polkadot-sdk/parachains/zero-to-hero/build-custom-pallet.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Build the Pallet
+description: Learn how to build a custom pallet for Polkadot SDK-based blockchains with this step-by-step guide. Create and configure a simple counter pallet from scratch.
+---
+
+# Build the Pallet
+
+## Introduction
+
+In Polkadot SDK-based blockchains, runtime functionality is built through modular components called [pallets](/polkadot-protocol/glossary#pallet){target=\_blank}. These pallets are Rust-based runtime modules created using [FRAME (Framework for Runtime Aggregation of Modular Entities)](/develop/parachains/customize-parachain/overview/){target=\_blank}, a powerful library that simplifies blockchain development by providing specialized macros and standardized patterns for building blockchain logic.
+A pallet encapsulates a specific set of blockchain functionalities, such as managing token balances, implementing governance mechanisms, or creating custom state transitions.
+
+In this tutorial, you'll learn how to create a custom pallet from scratch. You will develop a simple counter pallet with the following features:
+
+- Users can increment and decrement a counter
+- Only a [root origin](https://paritytech.github.io/polkadot-sdk/master/frame_system/pallet/type.Origin.html#variant.Root){target=\_blank} can set an arbitrary counter value
+
+You'll use the [Polkadot SDK Parachain Template](https://github.com/paritytech/polkadot-sdk/tree/master/templates/parachain){target=\_blank} set up in the previous tutorial.
```

Comment:
We can't assume users are following these tutorials sequentially, so please link to previous tutorial.